### ISSUE TITLE
Remove image_widget_crop/image_style_warmer workarounds once realtime queue runner is in place.

### ISSUE THREAD

**Issue by ndouglas** [department-of-veterans-affairs/va.gov-cms#4425]
## Description
#2743 is a result of a conflict in the hooks used by the contrib modules `image_style_warmer` and `image_widget_crop`.  `image_style_warmer` can operate at one of two times -- "initial," which is too early because it happens prior to the parent entity being saved, and "queue," which is too late because it only happens on cron runs. 

When we have a realtime queue runner in place, the "queue" setting should be perfectly acceptable.  This would mean that the underlying issue in #2743 is no longer a concern and the code changes introduced in #4348 are no longer necessary and can be safely undone.

## Acceptance Criteria
- [ ] the form alter and other hooks added as part of #4348 have been removed
- [ ] derivatives for manually cropped images are still generated appropriately as described in #2743
- [ ] no regressions are indicated by the behavioral tests added in #4348

**Comment by acrollet** [comment]
Closing in favor of new AC on #5601
